libraries/OpenThread/examples/COAP/coap_lamp/ci.json

```
@@ -0,0 +1,10 @@
+{
+    "targets": {
+      "esp32": false
+      "esp32c2": false
+      "esp32c3": false
+      "esp32s2": false
+      "esp32s3": false
+    }
+  }
```

@SuGlider CI is failing due to missing commas in the ci.json
```suggestion
{
    "targets": {
      "esp32": false,
      "esp32c2": false,
      "esp32c3": false,
      "esp32s2": false,
      "esp32s3": false
    }
  }
```

---

Thank @P-R-O-C-H-Y !!